examples/visualization/plot_ssp_projs_sensitivity_map.py

```diff
@@ -23,7 +24,9 @@
 fname = data_path + '/MEG/sample/sample_audvis-meg-eeg-oct-6-fwd.fif'
 ecg_fname = data_path + '/MEG/sample/sample_audvis_ecg-proj.fif'
 
-fwd = read_forward_solution(fname, surf_ori=True)
+fwd = read_forward_solution(fname)
+fwd = convert_forward_solution(fwd, surf_ori=True)
```

Comment:
Some or all of these tests will break once we change the default `use_cps` behavior, right? And many of these calls will give `FutureWarning` now?

Please change any that emit warnings to use explicit `use_cps=True` (preferably) or `use_cps=False` as necessary. This 1) avoids warnings in tests and examples for now and 2) future-protects us once we make the default change (and then we can also remove a bunch of `use_cps=True` calls).